parsl/executors/taskvine/executor.py

```diff
@@ -215,7 +216,8 @@ def __create_data_and_logging_dirs(self):
 
         # Create directories for data and results
         log_dir = os.path.join(run_dir, self.label)
-        self._function_data_dir = os.path.join(run_dir, self.label, "function_data")
+        tmp_dir = os.path.join('/tmp/', f'{self.label}-{os.getlogin()}')
+        self._function_data_dir = os.path.join(tmp_dir, datetime.now().strftime('%Y%m%d%H%M%S%f'), "function_data")
```

Comment:
This is more a reference for readers of #1122 that  review comment for @colinthomas-z80 - here's another instance of using a timestamp as a unique ID that #1122 might want to address